meta/and.rb

```diff
@@ -1,4 +1,4 @@
-Mutant::Meta::Example.add do
+Mutant::Meta::Example.add :and do
```

Comment:
Lets go full hardcore and parens, always. Easier to think. Lets discuss this in channel and think about shipping a base rubocop config in devtools?